pkg/controller/install/install_controller.go

```diff
@@ -176,37 +154,78 @@ func (r *ReconcileInstall) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
+	updateDomain(r.client, reqLogger)
+	updateServiceNetwork(r.client, reqLogger)
```

Comment:
Let's not pass the loggers as function params. Instead, create one from the global one in the same way `reqLogger` is created at the top of `Reconcile`